model/clusters_mgmt/v1/external_auth_type.model

```diff
@@ -84,7 +84,7 @@ struct ExternalAuthClaim {
 struct TokenClaimMappings {
 
     // Username is a name of the claim that should be used to construct usernames for the cluster identity.
-    UserName UsernameClaim
+    Username UsernameClaim
```

Comment:
You could also use the json annotation
```
 @json(name = "username")
 UserName UsernameClaim
```
This will ensure backward compatible for the `UserName` field, assuming that the sdk as already been release.

Reply:
I am not aware of anyone using this yet so it should be safe. But we can also change the json annotation if we want

Reply:
Yes. And if there were people using it, it's probably not working so we should be fine just changing it now. I wanted to share the above as mostly an FYI

Reply:
Thanks @machi1990 @andreadecorte , let me add the json annotation